SYCL/BFloat16/bf16_storage_builtins.cpp

```diff
@@ -0,0 +1,444 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out -Xsycl-target-backend --cuda-gpu-arch=sm_80
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out

```

Comment:
```suggestion
// RUN: %GPU_RUN_PLACEHOLDER %t.out

```
The comment below says that "// Only cuda backend implements bf16", so runnig this test on non-GPU devices doesn't make sense.